### DISCUSSION TITLE
Welcome to Avatar-Maker Discussions!

### DISCUSSION THREAD

**Discussion by favrora** [favrora/Avatar-Maker#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mofifoluwa-ade** [comment]
i love it

**Reply by favrora** [reply]
Thank! If you have any ideas for a project, we can discuss them here.

**Comment by xtareq** [comment]
Great app idea. we need to change it to vue3.

**Comment by musicdownload00** [comment]
Nice article, Thanks for sharing [picrew](https://picrew.to/)

**Comment by salonikoo** [comment]
Hey guys, looking good. big big like.
There’s any way I can customize and change the layers for my needs? I got properties in PNG files.
thank you

**Reply by favrora** [reply]
Hello @salonikoo. Thanks for the question. Can you send a link to an example?

**Reply by salonikoo** [reply]
> Hello @salonikoo. Thanks for the question. Can you send a link to an example?

Thank you for answering. I mean for all the layers properties, for example : Accessories, Clothes, Eyes, Glasses.. I want to replace with my files, my own hand-drawing layers, but those objects  are png files and not vue. Is that possible ?
link for example: https://www.sp-studio.de

**Reply by favrora** [reply]
@salonikoo I will add this feature in the future, thanks for asking. Now the priority is to rewrite the project on vue3, then we will start adding new features
